test-resources/clj-kondo.exports/marick/midje/marick/midje.clj

```diff
@@ -0,0 +1,23 @@
+(ns marick.midje
+  (:require [clj-kondo.hooks-api :as hooks]
+            [clojure.string :as string]))
+
+(defn table-variable? [node]
+  (let [sexpr (hooks/sexpr node)]
+    (and (symbol? sexpr)
+         (string/starts-with? (str sexpr) "?"))))
```

Comment:
the `?` as start is just convention. Determining table variables is a bit trickier. Because of that I introduced a syntax extension to tabular (for better or worse) that makes it clearer by allowing users to wrap the table variables in square brackets: https://github.com/marick/Midje/pull/392

Reply:
:+1: sounds like a good middle ground